MapboxCoreNavigation/RouteController.swift

```diff
@@ -106,8 +106,7 @@ open class RouteController: NSObject, Router {
      - parameter directions: The Directions object that created `route`.
      - parameter source: The data source for the RouteController.
      */
-    @objc(initWithRoute:directions:locationManager:)
-    public init(along route: Route, directions: Directions = Directions.shared, dataSource source: RouterDataSource) {
+    required public init(along route: Route, directions: Directions = Directions.shared, dataSource source: RouterDataSource) {
```

Comment:
This removes Objective-C compatibility, which is a breaking change as long as the initializer remains public.

Reply:
Oversight. There's no real reason why it can't be `@objc`.

Reply:
It bridges to Obj-C because the `Router` protocol is annotated with `@objc`